spec/default_module_facts.yml

```diff
@@ -4,3 +4,7 @@ python_dir: '/usr/local/lib/python2.7/dist-packages'
 operatingsystem: 'Debian'
 lsbdistid: 'Debian'
 lsbdistcodename: 'wheezy'
+os:
```

Comment:
This is, in my opinion, a bad idea. We should instead migrate the failing tests to rspec-puppet-facts. I will have a look at it and provide a PR